sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -83,9 +83,8 @@ object Klaviyo {
 
         Registry.get<ApiClient>().startService()
 
-        Registry.register<State>(KlaviyoState())
-        Registry.getOrNull<StateSideEffects>()?.detach()
-        Registry.register<StateSideEffects>(StateSideEffects())
+        Registry.registerOnce<State>(KlaviyoState())
+        Registry.registerOnce<StateSideEffects>(StateSideEffects())
```

Comment:
Registering these services once, instead of discarding and re-registering, solves the issue of other modules relying on analytics' `State` observers.